fineract-provider/src/main/java/org/apache/fineract/infrastructure/configuration/domain/ConfigurationDomainService.java

```diff
@@ -21,6 +21,7 @@
 import java.util.Date;
 import org.apache.fineract.infrastructure.cache.domain.CacheType;
 
+@SuppressWarnings({ "AbbreviationAsWordInName" })
```

Comment:
this is not test code, so it should be fixed.. is it because of `isSMSOTPDeliveryEnabled` etc.? Just make that e.g. `isSmsOtpDeliveryEnabled` - it's fine, and actually more not less readable (once you are used to the "convention" - and it's better to have it "uniform" across the entire code base).